android/src/main/java/com/smileidentity/react/expo/SmileIDDocumentVerificationEnhancedView.kt

```diff
@@ -30,7 +30,7 @@ class SmileIDDocumentVerificationEnhancedView(context: Context, appContext: AppC
     private val onResult by EventDispatcher()
     private val onError by EventDispatcher()
 
-    fun updateConfig(config: EnhancedDocumentVerificationRequest) {
+    fun updateConfig(config: EnhancedDocumentVerificationRecord) {
```

Comment:
I think `Request` suffix makes more sense than using `Record`

Reply:
I don't think so in this case - cause they're params and we're not making any request per se

Reply:
maybe call them `Params` then? since they are params

Reply:
Sure let me rename